### ISSUE TITLE
Fails to detect `E713 test for membership should be 'not in'` when left side more complex than simple variable

### ISSUE THREAD

**Issue by Kache** [PyCQA/pycodestyle#1212]
```py
var = {'foo': 3}

if not var in [1, 2, 3]:  # detected
    pass

# all the below are not detected

if not var['foo'] in [1, 2, 3]:
    pass

if not str(var) in [1, 2, 3]:
    pass

if not var.keys() in [1, 2, 3]:
    pass
```

**Comment by asottile** [comment]
please search the issue tracker next time. I've put in a bunch of work to make it easy to search by code
